.zenodo.json

```diff
@@ -36,71 +28,108 @@
       "orcid": "0000-0003-1761-2591"
     },
     {
-      "name": "Krude, Jakob",
-      "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf"
+      "name": "Stephan, Jan",
+      "affiliation": "CASUS, Helmholtz-Zentrum Dresden-Rossendorf",
+      "orcid": "0000-0001-7839-4386"
     },
     {
-      "name": "Rogers, David M.",
-      "affiliation": "Oak Ridge National Laboratory",
-      "orcid": "0000-0002-5187-1768"
+      "name": "Vyskočil, Jiří",
+      "affiliation":"CASUS, Helmholtz-Zentrum Dresden-Rossendorf",
+      "orcid": "0000-0001-8822-0929"
     },
     {
-      "affiliation": "CASUS, Helmholtz-Zentrum Dresden-Rossendorf",
-      "name": "Stephan, Jan",
-      "orcid": "0000-0001-7839-4386"
+      "name": "Widera, René",
+      "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf",
+      "orcid": "0000-0003-1642-0459"
+    }
+  ],
+  "contributors": [
+    {
+      "name": "Colgrove, Mat",
+      "affiliation": "NVIDIA",
+      "type": "Other"
     },
     {
       "name": "Gehrke, Valentin",
-      "affiliation": "TU Dresden"
+      "affiliation": "TU Dresden",
+      "type": "Other"
     },
     {
-      "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf, TU Dresden",
       "name": "Huebl, Axel",
-      "orcid": "0000-0003-1943-7141"
+      "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf, TU Dresden",
+      "orcid": "0000-0003-1943-7141",
+      "type": "Other"
     },
     {
       "name": "Knespel, Maximilian",
-      "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf"
+      "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf",
+      "type": "Other"
+    },
+    {
+      "name": "Krude, Jakob",
+      "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf",
+      "type": "Other"
     },
     {
       "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf, TU Dresden",
       "name": "Matthes, Alexander",
-      "orcid": "0000-0002-6702-2015"
+      "orcid": "0000-0002-6702-2015",
+      "type": "Other"
     },
     {
       "name": "Mewes, Hauke",
-      "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf"
+      "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf",
+      "type": "Other"
+    },
+    {
+      "name": "Nash, Phil",
```

Comment:
This feels weird. Is this an anonymous contributor or just someone contributing to us in his private time?

Reply:
He is the author of Catch2. He currently works at SonarSource.

Reply:
Okay, thanks for the hint. According to our git logs he has one commit from early 2019. In late 2019 he was working for JetBrains ([Source](https://www.youtube.com/watch?v=ZUH8p1EQswA)), so I'll put that in as affiliation.

Reply:
A further Google search shows that he joined JetBrains in 2016, so that should be a safe guess.